lib/helpbase.gi

```diff
@@ -1013,13 +1013,74 @@ InstallGlobalFunction(HELP_GET_MATCHES, function( books, topic, frombegin )
   return [exact, match];
 end);
 
+if not IsBound( InitialSubstringUTF8String ) then
+  InitialSubstringUTF8String:= "dummy";
+fi;
+if not IsBound( LETTERS ) then
+  LETTERS:= "dummy";
+fi;
+if not IsBound( WidthUTF8String ) then
+  WidthUTF8String:= "dummy";
+fi;
+
+BindGlobal( "InitialSubstringUTF8Text", function( str, cols )
```

Comment:
This is a bit confusing: there is function `InitialSubstringUTF8String` (from GAPDoc?) with an extremely similar name. What's the difference? Perhaps a code comment could explain that?

Reply:
Looking at the code, my guess is that `InitialSubstringUTF8String` takes an initial segment of `str` which fits into `cols` column in a terminal (taking multi byte encoded characters into account, perhaps "wide" Unicode characters or invisible ones like accent decorators) ?

And this function here "just" adds in handling of escape sequences? If so, perhaps the function name could reflect this better? InitialSubstringUTF8StringIgnoringEscapeSequences` rolls right off my tongue 😂 😝